solvers/UNet.py

```diff
@@ -0,0 +1,87 @@
+from benchopt import BaseSolver, safe_import_context, config
```

Comment:
please avoid capital letters in filenames :)

sorry I could not resist when I saw this ...